### PR TITLE
Datetime test cases

### LINKED ISSUE #61: Test Case for StartDate & EndDate

### PR DESCRIPTION
**Pull request by yirui94**
Fix #61

### CODE REVIEW

**Comment by VeryLazyBoy** [comment]
Can we combine StartDateTimeTest and EndDateTimeTest into DateTimeTest? @yirui94

**Comment by yirui94** [comment]
Okay no problem!

File: src/test/java/seedu/geekeep/logic/LogicManagerTest.java [hunk 1]
```diff
@@ -66,15 +66,45 @@
      */
     class TestDataHelper {
 
-        Task adam() throws Exception {
-            Title title = new Title("Adam Brown");
-            DateTime privateEndDateTime = new DateTime("01-05-17 1630");
+        Task event() throws Exception {
```
Comment (VeryLazyBoy) [inline src/test/java/seedu/geekeep/logic/LogicManagerTest.java]: Can we give  a  specific name like "inHospital"?

File: src/test/java/seedu/geekeep/logic/LogicManagerTest.java [hunk 2]
```diff
@@ -66,15 +66,45 @@
...
             DateTime startDateTime = new DateTime("01-04-17 1630");
-            Location privateLocation = new Location("111, alpha street");
+            Location location = new Location("111, alpha street");
             Tag tag1 = new Tag("tag1");
             Tag tag2 = new Tag("longertag2");
             UniqueTagList tags = new UniqueTagList(tag1, tag2);
-            return new Task(title, startDateTime, privateEndDateTime, privateLocation, tags, false);
+            return new Task(title, startDateTime, endDateTime, location, tags, false);
+        }
+
```
Comment (VeryLazyBoy) [inline src/test/java/seedu/geekeep/logic/LogicManagerTest.java]: assignment?

File: src/test/java/seedu/geekeep/logic/LogicManagerTest.java [hunk 3]
```diff
@@ -66,15 +66,45 @@
...
+            DateTime startDateTime = new DateTime("01-04-17");
+            Location location = new Location("111, alpha street");
+            Tag tag1 = new Tag("tag1");
+            Tag tag2 = new Tag("longertag2");
+            UniqueTagList tags = new UniqueTagList(tag1, tag2);
+            return new Task(title, startDateTime, endDateTime, location, tags, false);
+        }
+
+        Task floatingTask() throws Exception {
+            Title title = new Title("Floating Task");
```
Comment (VeryLazyBoy) [inline src/test/java/seedu/geekeep/logic/LogicManagerTest.java]: readBook?

**Review by VeryLazyBoy — COMMENTED** [review]
Minor things

File: src/test/java/seedu/geekeep/logic/LogicManagerTest.java (hunk 1 above)
Comment (yirui94) [reply src/test/java/seedu/geekeep/logic/LogicManagerTest.java]: Is it necessary though? We are mainly testing the dateTime over here. I think having the same title name as the method makes it clearer

**Pull request merged**